src/printer.rs

```diff
@@ -30,7 +26,7 @@ impl<'a> Printer<'a> {
         Printer {
             handle,
             colors,
-            term_width,
+            term_width: options.term_width,
```

Comment:
I think `term_width` can be removed from `Printer` if it is already included in `options`.